### PR TITLE
Add first version of comment system.

### PR DESCRIPTION
**Pull request by hasanbal**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -45,4 +30,22 @@ function popupDetail(id){
...
+}
+
+async function loadComments(){
+    const response = await fetch("/comments");
+    var comments = await response.text();
+    comments = JSON.parse(comments);
+
+    const commentsListElement = document.getElementById('comments');
+    commentsListElement.innerHTML = '';
+    for(var i=0;i<comments.length;i++){
```
Comment (maffina) [inline portfolio/src/main/webapp/script.js:48]: Please add whitespaces after ; and around +, = etc operators
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Comment (hasanbal) [reply portfolio/src/main/webapp/script.js]: I think, I added whitespaces in a proper way in my last commit.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,57 @@
...
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String comment = getComment(request);
+        boolean readable = false;
+        if (comment.isEmpty()) {
+            response.setContentType("text/html");
+            response.getWriter().println("Please enter a non-empty comment.");
+            return;
+        }
+        
+        for (int i=0; i < comment.length(); i++) {
```
Comment (maffina) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: whitespaces: "i = 0" here and later (as "=" is a binary operator)

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,37 +12,41 @@
...
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-
-function getRandomImage(){
+function getRandomImage() {
     const imageId = Math.floor(Math.random()*5) + 1;
```
Comment (maffina) [inline portfolio/src/main/webapp/script.js:16]: * is also a binary operator, please add whitespaces

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,37 +12,41 @@
...
     const imageTag = document.getElementById('randomImg');
 
-    imageTag.src = "images/ig"+imageId.toString()+".png";
+    imageTag.src = "images/ig" + imageId.toString() + ".png";
 }
 
-function popupDetail(id){
+function popupDetail(id) {
     const detail = document.getElementById('detail-'+id.toString());
```
Comment (maffina) [inline portfolio/src/main/webapp/script.js:23]: whitespaces around + here and the next line

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -12,37 +12,41 @@
...
+
+async function loadComments() {
+    const response = await fetch("/comments");
+    var comments = await response.text();
+    comments = JSON.parse(comments);
+
+    const commentsListElement = document.getElementById('comments');
+    commentsListElement.innerHTML = '';
+    
+    for (var i=0; i<comments.length; i++) {
```
Comment (maffina) [inline portfolio/src/main/webapp/script.js:49]: whitespaces: i = 0, i < comments.length, "Comment" + i

Comment (hasanbal) [reply portfolio/src/main/webapp/script.js]: Sorry for other mistakes. I hope linting will work after merging.

**Pull request merged**